ros/package.xml

```diff
@@ -44,6 +44,9 @@
   <exec_depend condition="$ROS_VERSION == 2">ros2launch</exec_depend>
 
   <!-- ROS1/2 dependencies -->
+  <depend>rcutils</depend>
```

Comment:
I guess this line can be moved to the "ROS 2 only" dependencies:
```xml
  <!-- ROS2 dependencies -->
  <buildtool_depend condition="$ROS_VERSION == 2">ament_cmake</buildtool_depend>
  <depend condition="$ROS_VERSION == 2">rclcpp</depend>
  <depend condition="$ROS_VERSION == 2">rclcpp_components</depend>
  <exec_depend condition="$ROS_VERSION == 2">ros2launch</exec_depend>
```

The other two rclcpp,  rclcpp_components are already there

Reply:
Ah, definetly not noticed them! I also saw the other comments, will reply / fix in the afternoon (in the meantime, one of my colleagues is testing this branch in a composed setup).